extensions/2.0/Khronos/KHR_image_formation/README.md

```diff
@@ -0,0 +1,197 @@
+
+Supporting this extension is only appropriate for rendering a single glTF scene; since it defines a global output mapping it is impossible to respect conflicting mappings for two glTFs in the same scene. Likewise, if a shop wanted a consistent “look” across  their portfolio, they might be better served by picking a consistent mapping function and requiring their artists to design with this mapping in mind, rather than displaying different products with unique parameters. 
+
+This mapping function is split into two parts: the linear multiplier, exposure, and the nonlinear look-up tables (LUT). Exposure represents what the human pupil does to compensate for scenes with different average light levels, but here we will define it in terms from the photography world, as they have already created standard physical units to represent the camera’s version of the pupil. 
+
+The non-linear LUT is much more complicated and arbitrary, but is nevertheless completely necessary. This compression step is necessary because the [0, infinity) range of physical light must be converted to the [0, 1] range of any output format. When no LUT is specified, this generally means the value is clamped, which is simply a piecewise linear compression function with sharp corners that  create unsightly artifacts. This is rarely the default mode for existing renderers, as it yields such poor results, but their  defaults are also rarely consistent with each other, hence the value of this extension.
+
+There is no “right” way to compress outgoing scene light, but the goal is to mimic the nonlinear response of the human retina without a lot of expensive computation. Camera sensors and film are likewise nonlinear and designed also around the human eye’s response. This is the purpose of the hdr_lut. Effectively we are mapping from physical light units to unitless perception values, represented as sRGB or other wide-gamut, HDR formats. 
+
+One key aspect of a good transform is that it does not operate solely on luminance but also affects color, especially saturation. This is because as a colored light becomes very bright, its color ceases to be perceived. This is true of CCDs as well, and can be easily seen by taking an overexposed photo of a bright, colored light and noting how it tends toward white. This is the purpose of the tone_mapping_lut.
+
+Regarding color gamuts, since glTF defines its textures in sRGB, so long as the scene’s lighting is also represented in Rec 709 (sRGB’s color gamut), then the output light will naturally also be restricted to the Rec 709 gamut. If a wide gamut like Rec 2020 is used for output, then the values will naturally fall into the Rec 709 subset. 
+
+## Extending Scene
+
+The precise method of converting scene linear light into output pixels is defined by adding the `KHR_image_formation` extension to any glTF scene. For example:
```

Comment:
I am worried that having a scene declaration that allows content creators to provide artistic intent may be confusing the format.
With this I mean the expected behavior of viewing a content creators model.
Imagine I create my model with a fancy tonemapping LUT that is Michael Bay'ish (saturated colors) and then this model is displayed in a scene with a distinct monochromatic LUT. This will happen, even if it's not part of the spec.
My model will now look totally different, and most probably wrong according to my artistic intent.

My worry is that users and content creators will see this as a serious flaw in glTF.
I suggest to remove the LUT's and focus on image formation.

Reply:
The LUT is a core part of how image formation is done in the photography, film, and CGI industries. It's used because it's important in making scenes look correct to the human eye despite the low range of printed or displayed images.

Reply:
> The LUT is a core part of how image formation is done in the photography, film, and CGI industries

Sure, and if the intention is to model the content creation pipeline of those industries I think this extension should be change to reflect that.

Reply:
Do you have a suggestion? I'm certainly not going to picky about names.

Reply:
My proposal would be to remove the LUTs with focus only on the aperture, shutterspeed and ISO parameters.
Making this into an extension that models a 'physical camera', suitable for studio photo/main product picture usecase.
At the same time making it compatible with KHR_displaymapping_pq, which could make it beneficial for broader use in 3D commerce.